pkg/buildclient/client.go

```diff
@@ -35,11 +35,11 @@ func Stream(ctx context.Context, cwd string, streams *streams.Output, dialer Web
 	conn, response, err := dialer(ctx, wsURL(build.Status.BuildURL), map[string][]string{
 		"X-Acorn-Build-Token": {build.Status.Token},
 	})
-	if response.Body != nil {
+	if response != nil {
```

Comment:
Should this be `response != nil && response.Body != nil`?

Reply:
I thought about that but it is not needed since `response.Body` is guaranteed to be not nil. Documentation around this [here](https://pkg.go.dev/net/http#Response:~:text=The%20http%20Client%20and%20Transport%20guarantee%20that%20Body%20is%20always%0A%09//%20non%2Dnil).